src/lib/y2packager/clients/inst_repositories_initialization.rb

```diff
@@ -101,6 +108,13 @@ def adjust_base_product_selection
       def products
         @products ||= Y2Packager::Product.available_base_products
       end
+
+      # Return the forced base product (if any)
+      #
+      # @return [Y2Product, nil]
+      def forced_base_product
+        @forced_base_product ||= Y2Packager::Product.forced_base_product
```

Comment:
The forced base product is already cached here
https://github.com/yast/yast-yast2/pull/954/files#diff-f69b205c192a907ae0fa6ae4feebaab1R125

Do we need to cache it again here? In the code above it would be enough to just use  
`forced_base_product = Y2Packager::Product.forced_base_product`

Reply:
You're right, I overlooked it :)

Thanks!